packages/terra-form-radio/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ ChangeLog
 
 Unreleased
 ----------
+### Changed
+* Adding a null check to the child components to avoid a crash
```

Comment:
nitpick - your change protects against all falsey values. Not just null.

Reply:
Updated: 0bf88b3a0